From d58c213c6d5c52ba44dcbbfe931a930268bb5e5b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alex=20R=C3=B8nne=20Petersen?= <alex@alexrp.com>
Date: Wed, 17 Sep 2025 00:25:21 +0200
Subject: [PATCH 075/133] test: re-enable test-std for loongarch64-linux
X-Developer-Signature: v=1; a=openpgp-sha256; l=1418; i=xtex@astrafall.org;
 h=from:subject; bh=CtC1waV4LWD0Tw1nMrzNE5dTGz+zGIyVoFLYL1MNM98=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn78li9Mvtm1d1gVpne4s1z8cF7bk75yItTJz16EPj2g
 Kro7cL+joksDGJcDJZiiixFhg3erDrp/KLLymVh5rAygQyRFmlgYGBgYGHgy03MKzXSMdIz1TbU
 MzTUMdIxYuDiFICpftzK8D8g1d5czvw+70JHbal3Xyy1Ps01iXoTw7GDp7MqPKS5VZ6RYcHGR0s
 f3FrR//Pc55/bPwm7fXU5oSjaFbDzcEv8FLk1FVwA
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit e06bcd74c3d7287dfb458130cde0c72c7ef97698)
---
 test/tests.zig | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/test/tests.zig b/test/tests.zig
index 637c7611ca39..3e5253bfc36b 100644
--- a/test/tests.zig
+++ b/test/tests.zig
@@ -424,8 +424,6 @@ const test_targets = blk: {
                 .os_tag = .linux,
                 .abi = .none,
             },
-            // https://github.com/ziglang/zig/issues/21646
-            .skip_modules = &.{"std"},
         },
         .{
             .target = .{
@@ -434,8 +432,6 @@ const test_targets = blk: {
                 .abi = .musl,
             },
             .link_libc = true,
-            // https://github.com/ziglang/zig/issues/21646
-            .skip_modules = &.{"std"},
         },
         .{
             .target = .{
@@ -445,8 +441,6 @@ const test_targets = blk: {
             },
             .linkage = .dynamic,
             .link_libc = true,
-            // https://github.com/ziglang/zig/issues/21646
-            .skip_modules = &.{"std"},
             .extra_target = true,
         },
         .{
@@ -456,8 +450,6 @@ const test_targets = blk: {
                 .abi = .gnu,
             },
             .link_libc = true,
-            // https://github.com/ziglang/zig/issues/21646
-            .skip_modules = &.{"std"},
         },
 
         .{
-- 
2.52.0

